pallets/ethereum-transactions/src/ethereum_transaction.rs

```diff
@@ -429,7 +434,7 @@ impl EthSignatures {
     }
 
     pub fn append(&mut self, mut other: Vec<ecdsa::Signature>) {
-        self.signatures_list.append(&mut other);
+        if let Err(_) = self.signatures_list.try_append(&mut other) {}
     }
```

Comment:
I think we shouldn't silently consume the error.
Normally I would suggest that we bubble up the error, but since this function is not called in the runtime, I would suggest we just delete it.